vochain/app.go

```diff
@@ -554,8 +579,8 @@ func (app *BaseApplication) Query(req abcitypes.RequestQuery) abcitypes.Response
 	return abcitypes.ResponseQuery{}
 }
 
-// EndBlock updates the app height and timestamp at the end of the current block
-func (app *BaseApplication) EndBlock(req abcitypes.RequestEndBlock) abcitypes.ResponseEndBlock {
+// fnEndBlockDefault updates the app height and timestamp at the end of the current block
+func (app *BaseApplication) fnEndBlockDefault(req abcitypes.RequestEndBlock) abcitypes.ResponseEndBlock {
```

Comment:
Why the `default` naming ?

Reply:
Do you have a better suggestion?

Reply:
I would just get rid of `default` and use `fnEndBlock` which is the default, and then you have the option of setting it for changing the behaviour

Reply:
This collides with the definition on BaseApplication

Reply:
ok, so ignore my comment. I do not have a better suggestion.